### PR TITLE
PR for cart management 

### PR DESCRIPTION
**Pull request by narharikale**
added following fuctionalities
- add to cart 
- remove from the cart
- move to wishlist
- price details

### CODE REVIEW

File: src/pages/cart/cart.jsx [hunk 1]
```diff
@@ -1,9 +1,46 @@
...
 function Cart() {
+ 
+    const { cartData } = useCart()
+
     return (  
-        <>
-        
-        </>
+            
+            <>
```
Comment (t007rushi) [inline src/pages/cart/cart.jsx:17]: no need ```<></>```

**Review by t007rushi — APPROVED** [review]
Looks Good to Merge

File: src/components/price-deatils-card/price-details-card.jsx [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+          <button className="btn btn-primary font-size-regular w-100-per">
+            Continue
+          </button>
+        </div>
+      </div>
+    </>
+  );
+}
+
+export { PriceDetailsCard };
```
Comment (horsemaker) [inline src/components/price-deatils-card/price-details-card.jsx:55]: You could maybe use some formatter to format your files.

File: src/context/cart-context.js [hunk 3]
```diff
@@ -0,0 +1,95 @@
+import axios from "axios";
+import { createContext, useContext } from "react";
+import { useState ,useEffect } from "react";
+import { useAuth } from "./auth-context";
+
+
+const cartContext = createContext(null);
+const useCart = () => useContext(cartContext);
+
+const authtoken = localStorage.getItem("AuthToken")
```
Comment (horsemaker) [inline src/context/cart-context.js]: You should move this inside the provider.

**Review by horsemaker — APPROVED** [review]

File: src/context/cart-context.js (hunk 3 above)
Comment (narharikale) [reply src/context/cart-context.js]: yes, fixed! thanks😀

File: src/components/price-deatils-card/price-details-card.jsx (hunk 2 above)
Comment (narharikale) [reply src/components/price-deatils-card/price-details-card.jsx:55]: yes , I'll configure that

File: src/pages/cart/cart.jsx (hunk 1 above)
Comment (narharikale) [reply src/pages/cart/cart.jsx:17]: yes , removed

**Pull request merged**